include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -432,6 +432,9 @@ typedef enum OrtMemType {
   OrtMemTypeDefault = 0,                // the default allocator for execution provider
 } OrtMemType;
 
+/**
+ * This API will be deprecated.
+*/
```

Comment:
Please provide information on what should be used instead.

Reply:
I'll remove this comment. The plan is, 1) deprecate this API, 2) users should call OrtCreateCpuAllocatorInfo such APIs to create allocatorinfo for different devices.

However, we only have one shortcut API for CPUAllocatorInfo right now. we'll add this deprecation comment when we have shortcuts for GPU.